docs/user_guide/remote_access_devices.rst

```diff
@@ -53,7 +53,7 @@ command line:
     $ python lewis-control.py device start
 
 Only numeric types and strings can be used as arguments via the
-``lewis-control.py``-script. The script always tries to convert
+``lewis-control``-script. The script always tries to convert
 parameters to ``int`` first, then to ``float`` and leaves it as ``str``
 if both fail. For other types and more control over types, it's advised
```

Comment:
I don't think this statement is accurate anymore. We now use `ast.literal_eval`, which should support any kind of literal.